app/src/entry-point.lisp

```diff
@@ -10,7 +10,7 @@
 (defvar *entered-from-main* nil)
 (defvar *program-name* "qvm")
 
-(defparameter *benchmark-types* '("bell" "qft" "hadamard" "qualcs" #-forest-sdk "suite")
+(defparameter *benchmark-types* '("bell" "qft" "hadamard" "qualcs" "baseline" #-forest-sdk "suite")
```

Comment:
Does this need adding to the option description also.

Reply:
does that automatically